code/frameworks/nextjs/README.md

```diff
@@ -232,44 +366,51 @@ The default values on the stubbed router are as follows (see [globals](https://s
 
 ```ts
 const defaultRouter = {
-  locale: context?.globals?.locale,
-  route: '/',
-  pathname: '/',
-  query: {},
-  asPath: '/',
-  push(...args: unknown[]) {
+  push(...args) {
     action('nextRouter.push')(...args);
     return Promise.resolve(true);
   },
-  replace(...args: unknown[]) {
+  replace(...args) {
     action('nextRouter.replace')(...args);
     return Promise.resolve(true);
   },
-  reload(...args: unknown[]) {
+  reload(...args) {
     action('nextRouter.reload')(...args);
   },
-  back(...args: unknown[]) {
+  back(...args) {
     action('nextRouter.back')(...args);
   },
-  prefetch(...args: unknown[]) {
+  forward() {
+    action('nextRouter.forward')();
+  },
+  prefetch(...args) {
     action('nextRouter.prefetch')(...args);
     return Promise.resolve();
   },
-  beforePopState(...args: unknown[]) {
+  beforePopState(...args) {
     action('nextRouter.beforePopState')(...args);
   },
   events: {
-    on(...args: unknown[]) {
+    on(...args) {
       action('nextRouter.events.on')(...args);
     },
-    off(...args: unknown[]) {
+    off(...args) {
       action('nextRouter.events.off')(...args);
     },
-    emit(...args: unknown[]) {
+    emit(...args) {
       action('nextRouter.events.emit')(...args);
     },
   },
+  locale: globals?.locale,
```

Comment:
Where do I get access to `globals`? Is this code running in a decorator? Or in the story function? Or ... ?

Reply:
Ok I see the code below & how it works. But I don't think this is super clear for the reader unless they go read the code. Perhaps something like:

```
The default router has access to `args` and `globals` in its local scope and is configured as follows:

....

As a user if you'd like to make use of `args` and `globals` to define the routing behavior you can ... (I don't actually know how this would look)
```